caravel/controllers/listings.py

```diff
@@ -18,6 +18,14 @@
 def search_listings():
     """Display a list of listings that match the given query."""
 
+    # Fix session handler if not initialized
+    if not session.get("view"):
```

Comment:
Oh right- I mean, the "?key=value" thingies at the end of the URL.

Since sessions are shared globally (across a browsing context), rather than per-tab, setting the view in the session sets it for all tabs, and so might cause wonkage when using the view continuation logic.